controllers/certificatepolicy_utils.go

```diff
@@ -112,6 +112,8 @@ func convertPolicyStatusToString(plc *policyv1.CertificatePolicy, defaultDuratio
 					return fmt.Sprintf(format, result, "No namespaces matched the namespace selector.")
 				}
 			}
+
+			return fmt.Sprintf(format, result, "All evaluated certificates are compliant")
```

Comment:
Is there a reason not to include this outside of `if len(plc.Status.CompliancyDetails) == 1 {`?